packages/gatsby/src/query/file-parser.ts

```diff
@@ -393,12 +432,13 @@ async function findGraphQLTags(
 
 const cache = {}
 
-export default class FileParser {
-  constructor({ parentSpan } = {}) {
-    this.parentSpan = parentSpan
-  }
+export class FileParser {
+  constructor(private parentSpan?: object) {}
```

Comment:
`Span` from opentracing

Reply:
Can you revert to the old syntax with a destructured object. I'd rather avoid changing APIs without a good reason.